packages/components/src/circular-progress-bar/index.tsx

```diff
@@ -1,4 +1,6 @@
 import clsx from 'clsx';
+import type { ReactNode } from 'react';
+
 import './style.scss';
 
 const CircularProgressBar = ( {
```

Comment:
This is such a simple self-contained component. I'd be tempted to copy-paste it into the dashboard code. Thus removing the need to specially allow imports of this `@automattic/components` export.

In fact I believe we did use to have our own donut component in the dashboard code. One of the old overview cards had a circular progress bar. It must have been deleted when those old cards were removed.

We'll want to follow this DS rykv85Ii9RkvP504dtqqaW-fi-158_52165 for the `CircularProgress`. If there are changes that happen to this component that are only for Calypso, we don't want 'em.